hydrolib/core/io/friction/models.py

```diff
@@ -1,18 +1,38 @@
 import logging
+from enum import Enum
 from pathlib import Path
 from typing import List, Literal, Optional
 
 from pydantic import Field, NonNegativeInt, PositiveInt
 from pydantic.class_validators import validator
 
 from hydrolib.core.io.ini.models import INIBasedModel, INIGeneral, INIModel
-from hydrolib.core.io.ini.util import get_split_string_on_delimiter_validator
-
-from ..ini.util import make_list_validator
+from hydrolib.core.io.ini.util import (
+    get_enum_validator,
+    get_split_string_on_delimiter_validator,
+    make_list_validator,
+)
 
 logger = logging.getLogger(__name__)
 
 
+class FrictionType(str, Enum):
+    """
+    Enum class containing the valid values for the frictionType
+    attribute in several subclasses of Structure/CrossSection/friction.models.
+    """
+
+    chezy = "Chezy"  # Chézy C [m 1/2 /s]
```

Comment:
These comments might be better placed in the docstring?

Reply:
Done, as postfix, now nice apidocs generated.